### PR TITLE
[ADD] Add support for swipe between form records on mobile devices

### PR DESCRIPTION
**Pull request by wpichler**

### CODE REVIEW

**Review by gonzaloruzafa — COMMENTED** [review]
Hi, tried to swipe between forms on iphone 6 SE with runbot instance, could not make it work, any suggestion? I am reviewing

**Comment by wpichler** [comment]
please try the used js lib directly here: http://hammerjs.github.io/#try-it - does it work on iPhone.

I do not own an iphone - so not tested with iphone - but it should work

**Comment by wpichler** [comment]
i've checked the runbot installation - module is not installed there - and not available.

How to fix this ?

File: web_gestures/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (wtaferner) [inline web_gestures/__init__.py]: Editor specific stuff should be removed

File: web_gestures/static/src/js/gestures.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+odoo.define('web_gestures', function (require) {
```
Comment (wtaferner) [inline web_gestures/static/src/js/gestures.js]: The whole module should be named in singular, please rename it to ```web_gesture```

**Review by wtaferner — CHANGES_REQUESTED** [review]
Nice contribution IMO, the reason for not working on runbot is the missing ```__manifest__.py```

File: web_gestures/__init__.py (hunk 1 above)
Comment (wpichler) [reply web_gestures/__init__.py]: done

File: web_gestures/static/src/js/gestures.js (hunk 2 above)
Comment (wpichler) [reply web_gestures/static/src/js/gestures.js]: done

**Pull request closed without merging**